### ISSUE TITLE
Unique note skins for specific player or opponent (for Psych v1.0)

### ISSUE THREAD

**Issue by XRed16** [ShadowMario/FNF-PsychEngine#15765]
### Describe your problem here.

I made a LUA script that intends to change the default noteskins to different textured noteskins for specific characters in the song.

While making this script taking inspirations from other questions and Gamebanana scripts, I ran into several issues that prevent me from achieving what I want the script to do.

This is the script that I made:
[Noteskins Issue - EXAMPLE SCRIPT.zip](https://github.com/user-attachments/files/17528911/Noteskins.Issue.-.EXAMPLE.SCRIPT.zip)

The script goes something like this:

If dadName equals 'dad' or 'darnell' then change the note skin. I tried to find a way to simplify the if conditions and it didn't work, so I put multiple if conditions.  The note change was done through a separate function. Unfortunately, when the note texture changes, the position of the notes are misplaced if the note texture is Chip or Future. If something like a Change Character event occurs and once it happens and checks if the dadName or boyfriendName is included, and if not, changes it back to the default note skin and note splashes.

Also, the note textures also overrides the texture to notetypes including custom ones and I could not find a way to replace the normal note textures without changing the texture of other notetypes. I also had to change the 'unspawnNotes' to 'notes' because some notes that appear sometimes use the default note texture. I have a feeling that some note textures might not change in certain sections with a lot of notes. 

Finally, I could not find a way to code how to change back to default note skin or note splash in a more simplified way no matter what note skin is selected like Default, Chip, or Future, and note splashes like Default, Vanilla, or Electric. getProperty will have to be used here.

TL;DR
- I made a Noteskin script. I made multiple if conditions that leads to changing noteskin texture.
- Sometimes, custom noteskins texture appears with incorrect offsets.
- Can't find a way to change noteskin texture WITHOUT changing the custom notetype textures.
- Couldn't find out how to check if it is not a character, then change noteskin back to default selected normal note skin and note splash. Use one of the getProperty commands here.
- Is there a separate variable that I could use like: setProperty('noteSkin', 'noteSkins/CUSTOMNOTESKIN')

### Are you modding a build from source or with Lua?

Lua

### What is your build target?

Windows x64

### Did you edit anything in this build? If so, mention or summarize your changes.

No, but is it possible to do this on a .hx script?
